tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "target": "es5",
```

Comment:
Why do you need `es5` as target while having `electron 8.1.1` as peer dependency? It's not like that `electron-reload` is to be run inside a browser 😄

Reply:
Thank you for the great feedback! Not sure which ES version that node and electron target, I bumped the compile target to es2015 for now.